RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

```diff
@@ -24,7 +26,9 @@
     "drop recoPFClusters_*_*_*",
     "keep recoElectronSeeds_*_*_*",
     "keep recoGsfElectrons_*_*_*",
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*'
+    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
+    'keep recoPhotons_gedPhotonsTmp_*_*',
```

Comment:
Ok, and when I add this relinking I will remove these 'keep' statements from the event content, as the non-Tmp collections will be saved, since the HI event content is a superset of the pp one.